src/style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.form-search {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search-icon-big {
+  border: none;
+  display: block;
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  background-color: transparent;
+}
+
+.search-icon-small {
+  border: none;
+  display: block;
+  position: absolute;
+  background-color: transparent;
+  left: 10px;
+  top: 10px;
+}
+
+.small-img {
+  width: 11px;
+  height: 11px;
+}
+
+.search-input {
+  display: block;
+  width: 100%;
+  outline: none;
+  font-family: inherit;
+  font-weight: 900;
+  color: rgb(61, 78, 97);
+  border: 1px solid rgb(225, 231, 237);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  background-repeat: no-repeat;
+}
+
+.big_input_search {
+  height: 70px;
+  line-height: 22px;
+  font-size: 16px;
+  box-sizing: border-box;
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: 26px 25px;
+  padding: 25px 0 26px 62px;
+  margin: 20px 0;
+}
+
+.big_input_search::placeholder {
+  text-shadow: none;
+  color: rgb(61, 78, 97);
+  font-size: 16px;
+  font-family: inherit;
+  font-weight: 300;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.small_input_search::placeholder {
+  font-family: inherit;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 19px;
+  color: rgb(61, 78, 97);
+  text-shadow: none;
+}
+
+.small_input_search {
+  font-size: 14px;
+  line-height: 19px;
+  height: 42px;
+  box-sizing: border-box;
+  padding-left: 33px;
+  background-size: 11px;
+  background-position: 13px 15px;
+}
+
+.big_input_search:hover,
+.small_input_search:hover {
```

Comment:
You can use common class here to style big and small input at once **search-input**